src/components/pages/membership_form/subpages/PaymentPage.vue

```diff
@@ -96,7 +96,7 @@ const next = async (): Promise<any> => {
 			storeCleanupActions.push( store.dispatch( action( NS_BANKDATA, markemptyBankDataValuesAsInvalid ) ) );
 		}
 		return Promise.all( storeCleanupActions ).then( () => {
-			if ( store.getters.paymentDataIsValid && store.getters[ NS_MEMBERSHIP_ADDRESS + '/membershipTypeIsValid' ] ) {
+			if ( store.getters[ NS_MEMBERSHIP_FEE + '/paymentDataIsValid' ] && store.getters[ NS_MEMBERSHIP_ADDRESS + '/membershipTypeIsValid' ] ) {
```

Comment:
Please undo this or inline the `paymentDataIsValid` from the store (and remove it), otherwise the bank data won't be validated.